### ISSUE TITLE
Right Shortcut? 

### ISSUE THREAD

**Issue by rashad** [Booligoosh/makerlog-menubar#4]
On PH page 👉 cmd+M - actually minimizes the window 😅 
On official page 👉 shit+M - actually types a capital M in whatever is opened 😂 

Might be linked to [Bartender](https://www.macbartender.com/)? Any idea?

**Comment by Booligoosh** [comment]
It's cmd + shift + M 😁

**Comment by Booligoosh** [comment]
Tell me if it works!

**Comment by m1guelpf** [comment]
It does (for me)!

**Comment by rashad** [comment]
Thanks for your answer @Booligoosh !! At first, I was surprised since the strict cmd + shift + M shortcut did not work. Then I figured that since it works for @m1guelpf it has to work for me. Since I'm in France my keyboard is not a qwerty but rather an azerty and guess what, the right shortcut for people using and azerty keyboard is cmd + maj + `,` (where the m  is located on qwerty keybords 😅 

**TL;DR: azerty keyboard are set to `cmd + shift + ,`**

Thanks for your help guys 👍
